core/chains/evm/logpoller/log_poller.go

```diff
@@ -59,6 +59,7 @@ type LogPoller interface {
 	IndexedLogsWithSigsExcluding(address common.Address, eventSigA, eventSigB common.Hash, topicIndex int, fromBlock, toBlock int64, confs Confirmations, qopts ...pg.QOpt) ([]Log, error)
 	LogsDataWordRange(eventSig common.Hash, address common.Address, wordIndex int, wordValueMin, wordValueMax common.Hash, confs Confirmations, qopts ...pg.QOpt) ([]Log, error)
 	LogsDataWordGreaterThan(eventSig common.Hash, address common.Address, wordIndex int, wordValueMin common.Hash, confs Confirmations, qopts ...pg.QOpt) ([]Log, error)
+	LogsDataWordBetween(eventSig common.Hash, address common.Address, wordIndexMin, wordIndexMax int, wordValue common.Hash, confs Confirmations, qopts ...pg.QOpt) ([]Log, error)
```

Comment:
This reminds me, we should really be updating the dev guide for LogPoller as well whenever we change the API:
https://github.com/smartcontractkit/chainlink-blueprints/blob/main/relay/evm/logpoller/api.md

At this point I think it's already gotten a bit out of date, but I'd like us to get in the habit of updating both at once.

Reply:
I think better to merge em.  We can just document the methods here then use godoc to generate it. Could be a periodic CI job in the blueprints repo

Reply:
Sounds reasonable, it would resolve all the issues with the consistency between these two repos